packages/bech32m/src/penumbra.ts

```diff
@@ -0,0 +1,15 @@
+import { fromBech32m, toBech32m } from './format/convert';
+import { Inner } from './format/inner';
+import { Prefixes } from './format/prefix';
+
+const innerName = Inner.penumbra;
+const prefix = Prefixes.penumbra;
+
+export const bech32mAddress = ({ [innerName]: bytes }: { [innerName]: Uint8Array }) =>
+  toBech32m(bytes, prefix);
+
+export const addressFromBech32m = (penumbra: string): { [innerName]: Uint8Array } => ({
```

Comment:
Realizing this feedback can apply to all of the different asset types in this package

Reply:
the reason for not doing this is described in the readme. we must do it in many places because many of our parameters and props are typed incorrectly. i would prefer to correct those instead but that is large

Reply:
> Can we rename param to `(penumbraAddr: string)`?

all of these to/from function parameters are named after the prefix of the expected input

Reply:
now these are named like `addressFromBech32m(penumbra1: string)`